CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This file is used to list changes made in each version of the openssh cookbook.
 
 ## Unreleased
 
+- Sous Chefs Adoption
+- Cookstyle fixes
+- Standardise files with files in sous-chefs/repo-management
 - resolved cookstyle error: libraries/helpers.rb:81:16 warning: `Lint/SendWithMixinArgument`
```

Comment:
I've been rewriting these in a condensed form and just have `Cookstyle fixes` to help with how this gets displayed in the announcement channel.

Reply:
Let me clarify, I **replace** the automated messages with *just* `Cookstyle fixes`. Can you please make that change here?

Reply:
ah, derp